components/list/list.css

```diff
@@ -0,0 +1,50 @@
+.horizontal-list{
+    padding: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+    
+}
+
+.list-width{
+    width: 25rem;
+}
+
+.list li{
+    padding: 1rem 2rem;
+    background-color: var(--light-bg);
+    list-style-type: none;
+    /* display: inline-block; */
```

Comment:
Just a suggestion 
remove comment if not in use

Reply:
I will remove this , thank you for review